src/app/screens/History/components/NoteItem/index.js

```diff
@@ -1,56 +1,61 @@
 import React, { useState } from 'react';
 import { string, shape, number } from 'prop-types';
-import { useDispatch } from 'react-redux';
+import { useDispatch, useSelector } from 'react-redux';
 import i18 from 'i18next';
 
-import useToastContext from 'hooks/useToastContext';
-import { handleTextStyle, countWords } from 'utils/functionUtils';
+import { countWords } from 'utils/functionUtils';
 import InlineInput from 'app/components/InlineInput';
 import InlineTextArea from 'app/components/InlineTextArea';
+import useToastContext from 'utils/hooks/useToastContext';
 import NotepadButton from 'app/components/NotepadButton';
 import NotesActions from 'redux/notes/actions';
 
 import styles from './styles.module.scss';
 
 const NoteItem = ({ note }) => {
   const [titleValue, setTitleValue] = useState(note.title);
-  const [textValue, setTextValue] = useState(note.text);
-  const [textClassNames, setTextClassNames] = useState([]);
+  const [textValue, setTextValue] = useState(note.content);
+  const [italic, setItalic] = useState(note.italic);
+  const [bold, setBold] = useState(note.bold);
   const [isTitleOpen, setIsTitleOpen] = useState(false);
 
   const addToast = useToastContext();
 
   const dispatch = useDispatch();
 
+  const deleteNoteLoading = useSelector(state => state.notes.deleteNoteLoading);
```

Comment:
move this to the bottom of the file

```
import { createSelector } from 'reselect';
...
export const loadingSelector = createSelector(
  store => store.notes.deleteNoteLoading,
  (deleteNoteLoading) => deleteNoteLoading
);


const mapStateToProps = state => ({
  loading: loadingSelector(state)
});
```
anyways, as we have 1 loading variable, a selector is not necessary here